homeassistant/components/velux/strings.json

```
@@ -33,6 +33,16 @@
       "rain_sensor": {
         "name": "Rain sensor"
       }
+    },
+    "select": {
+      "cover_velocity": {
+        "name": "Cover velocity",
```

Sorry, I meant the name of the select:

```suggestion
        "name": "Open/Close velocity",
```